modules/claims_api/app/controllers/claims_api/v0/forms/disability_compensation_controller.rb

```diff
@@ -23,6 +23,7 @@ def submit_form_526
             status: ClaimsApi::AutoEstablishedClaim::PENDING,
             auth_headers: auth_headers,
             form_data: form_attributes,
+            flashes: form_attributes.dig('veteran', 'flashes') || [],
```

Comment:
This logic is shared between v0 and v1 so makes sense to have a "flashes" method in a shared parent class of these 2.
Breaking out into a method also makes sense because this logic needs to be a bit more complicated based on what is in master currently:
https://github.com/department-of-veterans-affairs/vets-api/blob/4150d82176fda1dd3f42b3687bf534c2cbf8fd96/modules/claims_api/app/models/claims_api/auto_established_claim.rb#L85

Should hopefully just be a matter of moving that flashes method out of the model and into the controller.